bin/sce_qc_report.R

```diff
@@ -14,6 +16,13 @@ option_list <- list(
     default = NULL,
     help = "path to rmd template file for report"
   ),
+  make_option(
+    opt_str = c("--celltype_report_template"),
+    type = "character",
```

Comment:
I like defaults, even when they don't really matter.
```suggestion
    type = "character",
    default = NULL,
```